### ISSUE TITLE
Please provide a descriptive title here, then fill out the rest of the template.

### ISSUE THREAD

**Issue by Kennan2002** [akeaswaran/cfc-bugs#38]
Please fill out the template below to make sure I can assist you most effectively.

**Describe the bug**

**To Reproduce**

**iOS version**: Version of iOS this bug occurred on (can be found in Settings.app > General)
**CFC version**: Version of CFC this bug occurred on (can be found in CFC's My Team/Career tab > Gear icon > scroll to the bottom)
**iOS device**: For example iPhone XS, iPad mini, etc.
**Jailbroken**: Whether or not your device is jailbroken
**Reproducible:** Can you reproduce it every time? Or is it random?

**Expected behavior**

**Screenshots**
If possible, add screenshots to help show the problem.

**Video**
If possible, attach a video to help show the problem. You can attach a video by uploading it to Streamable, Gfycat, YouTube, etc. and pasting the URL here.

**Additional context**
Add more teams like Clemson Penn State TCU UCF Texas Tech Notre Dame Miami Stanford USC FAU FIU Rutgers Boise State Baylor SMU Virginia Tech West Virginia Marshall Ole Miss Mississippi St Arizona ST Arizona Purdue Louisiana Tech Georgia Georgia Tech Iowa Iowa St South Carolina UNC Auburn FSU Boston college Texas A&M UCLA Syracuse NC state Arkansas Nebraska BYU
Navy Army- make them an annual bowl game Missouri Vanderbilt

Arrange conferences by prestige not just location 
Have subdivisions with teams able to get 5 star recruits but on the lower end 
Add some more school like American Samoa to and make them into a conference 

Add special teams kick and punt returner 

Add coordinators who can add boost to your schemes but also your player example: Offensive coordinator who runs a spread add 1+ long pass play and boost receivers 1+ overall

Add recieving yards for running back 

Make ol into Tackle Guard and Center

Make DL into De DT NT(NT schools get boost in 3-4 defense)

Add more awards like best receiver in the nation
Or Senior of the year change all league to all American 

A problem when computer selects all league player I’ll have 3 receivers who all get around 150+ catches 2500-3000 yards recieving and 30+tds and only one makes the all league team while the other receivers who make the all league team don’t even have 1000 yards recieving or double digit tds pls fix ASAP 

Have Info on high school recruits like all state selection Or state champion winner add more 5 star recruitS per class

Fix smash mouth offensive where running back get realistic numbers- I’ll have a running back get 800 yds on 400 Carries but in a spread which is a pass heavy offense get 1800 yards on 250 carries the running backs were both high 90s and both offensive line had no on lower than a 90 pls fix immediately 

Add ol to all league and conference teams
Add sacks allowed By ol

Add clutch trait makes players play better while at home or in big moments 

Make coty of the year more realistic I’ll go 15-0 have the 1 offensive and defense average 40+ pats  game allow 2pt per game and lose to a sock that went 12-2 I understand the prestige of the Program does come into play but pls try to even it out 

Fix recruiting news I’ll get the most recruits and they will all be 5 star top 20 players and still not make the top 5 recruiting class pls fix

Add fullback to smashmouth offense 

When recruiting only show player that are available for the programs prestige. When I’m playing as American Samoa it’s annoying scrolling down to find the top player I can recruit 
Make the convicne early draft declares like the reciting class you get 4 ways to convince and get a chance to flip their decision but this a little effort than normal recruiting 

Make home and away games a factor when it come to the win lose probality 

Add Deffensive player of the year

**Comment by akeaswaran** [comment]
Thanks for all of this feedback! Next time, please be sure to separate each item properly depending on if it's a bug or a feature request -- a lot of stuff is mixed in here and it's a little hard to parse through. I'll try my best to go through each and every part here:

> Add more teams like Clemson Penn State TCU UCF Texas Tech Notre Dame Miami Stanford USC FAU FIU Rutgers Boise State Baylor SMU Virginia Tech West Virginia Marshall Ole Miss Mississippi St Arizona ST Arizona Purdue Louisiana Tech Georgia Georgia Tech Iowa Iowa St South Carolina UNC Auburn FSU Boston college Texas A&M UCLA Syracuse NC state Arkansas Nebraska BYU
Navy Army- make them an annual bowl game Missouri Vanderbilt

I can't do this because these names are all trademarked. It would take a lot of money that I do not have to pay for licensing fees, and I would probably not be able to get those licensing fees anyway. Using state names itself is a bit of a gray area that provides plausible deniability.

> Arrange conferences by prestige not just location

This isn't necessarily how it works in real life.

> Have subdivisions with teams able to get 5 star recruits but on the lower end

I'm not exactly sure what this means -- can you please provide some more detail?

> Add coordinators who can add boost to your schemes but also your player example: Offensive coordinator who runs a spread add 1+ long pass play and boost receivers 1+ overall
> Add recieving yards for running back
> Add Deffensive player of the year

I'm considering these for a future release (but no promises).

> Add special teams kick and punt returner
> Make ol into Tackle Guard and Center
> Make DL into De DT NT(NT schools get boost in 3-4 defense)

The game doesn't really consider individual lineman play, so these designations would be aesthetic only. Given that, I don't really see the need for them -- but I'm always open to feedback!

> Add more awards like best receiver in the nation

Considering this in a future release, but again -- no promises.

> change all league to all American

Can't do this either -- see explanation about real team names.

> A problem when computer selects all league player I’ll have 3 receivers who all get around 150+ catches 2500-3000 yards recieving and 30+tds and only one makes the all league team while the other receivers who make the all league team don’t even have 1000 yards recieving or double digit tds pls fix ASAP

I'll look into this when I get the chance.

> Add ol to all league and conference teams
> Add sacks allowed By ol

These don't really make sense given the current state of the game engine -- the OL is considered as a unit, rather than as individuals. 

> Fix smash mouth offensive where running back get realistic numbers- I’ll have a running back get 800 yds on 400 Carries but in a spread which is a pass heavy offense get 1800 yards on 250 carries the running backs were both high 90s and both offensive line had no on lower than a 90 pls fix immediately

There are engine balancing changes coming in a future release. They're tricky to get right so I haven't prioritized them quite yet, but rest assured: they're coming.

> Add clutch trait makes players play better while at home or in big moments

There's already something like this in the engine, but I'll see what else I can do.

> Make coty of the year more realistic I’ll go 15-0 have the 1 offensive and defense average 40+ pats game allow 2pt per game and lose to a sock that went 12-2 I understand the prestige of the Program does come into play but pls try to even it out

This seems fine to me and is pretty realistic because the main factor in deciding a COTY is prestige increase, not current prestige. If your program is winning all the time, everyone already knows you're a good coach -- you shouldn't be raking in the COTY awards (See: LeBron James and the NBA MVP award). 

> Fix recruiting news I’ll get the most recruits and they will all be 5 star top 20 players and still not make the top 5 recruiting class pls fix

I'll take a look at this when I get the chance.

> Add fullback to smashmouth offense

I am not adding more positions to the game at this time -- I don't want to make it a large unwieldly spreadsheet with a lot of buttons and knobs to mess with.

> When recruiting only show player that are available for the programs prestige. When I’m playing as American Samoa it’s annoying scrolling down to find the top player I can recruit

This feature used to be in the game but was cut because offering sorts was a much better option. I encourage you to use the sorts to sort by interest to find the right players. Change to make these sorts more robust will be coming in a future release.

> Make the convicne early draft declares like the reciting class you get 4 ways to convince and get a chance to flip their decision but this a little effort than normal recruiting

Eh, I don't necessarily think this is how this process works in real life. There's no action you can really take as a coach to convince a really good player to come back to campus; they're already going to get a lot of playing time, they already know the coaches, etc. It's just a conversation.

> Make home and away games a factor when it come to the win lose probality

Win probability is already adjusted for home-field advantage.
